docs/source/getting-started.rst

```diff
@@ -121,17 +126,25 @@ SNS for both sending the log data and receiving the alert but StreamAlert also s
   aws sns create-topic --name streamalert-test-data
   aws sns create-topic --name streamalert-test-alerts
 
-2. Subscribe your email to the alerts SNS topic (replacing REGION, ACCOUNTID, and EMAIL with your own values):
+2. Export some environment variables for easy re-use later:
+
+.. code-block:: bash
+
+  export SA_REGION=us-east-1        # StreamAlert deployment region
+  export SA_ACCOUNT=123412341234    # AWS account ID
```

Comment:
Do you change the account ID to `111111111111` which is used as example at line 93.